korangar/src/settings/graphic.rs

```diff
@@ -29,13 +29,13 @@ impl Default for GraphicsSettings {
             lighting_mode: LightingMode::Enhanced,
             vsync: true,
             limit_framerate: LimitFramerate::Unlimited,
-            triple_buffering: true,
+            triple_buffering: false,
```

Comment:
Is this change intentional?

Reply:
No. Must be a leftover of me testing a lot.